pages/mesosphere/dcos/services/kubernetes/2.6.1-1.17.8/release-notes/index.md

```diff
@@ -27,9 +27,9 @@ excerpt: Release notes for DC/OS Kubernetes version 2.6.1-1.17.8
 * Kubernetes v1.17.8
 * Docker v19.03.11
 
-## Bug Fixes
+## Fixed Issues
```

Comment:
This is inconsistent with the other release notes on the site (e.g. https://docs.d2iq.com/mesosphere/dcos/services/kubernetes/2.6.0-1.17.7/release-notes/#bug-fixes from previous release). Also, would you be able to ping me in future to make these changes so I can ensure that any changes made in this docs site are backported to the dcos-kubernetes-cluster package repo (as the authoritative source) so we don't accidentally overwrite these changes in future when we pull docs in?